### PR TITLE
HT sidesheet and garden custom group view for HT

### PR DESCRIPTION
**Pull request by Andreas-Tonnessen**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: [AB#201224](https://dev.azure.com/Equinor/fa63ceea-8883-41e2-8823-a3b54e4be178/_workitems/edit/201224)

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/DisciplineReleaseControl/Components/Garden/ReleaseControlGardenGroupView.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+const ReleaseControlGardenGroupView = ({
+    data,
+    columnExpanded,
+    onClick,
+    onSelect,
+}: CustomGroupView<Pipetest>) => {
+    const handleSubGroupClick = () => {
+        if (data.value.startsWith('HT')) {
+            onSelect && onSelect(data as any);
```
Comment (jaysencpp) [inline src/apps/DisciplineReleaseControl/Components/Garden/ReleaseControlGardenGroupView.tsx:22]: må det være as any her?

File: src/apps/DisciplineReleaseControl/Components/Sidesheet/ReleaseControlHTSidesheet.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import { ServerError } from '../../Api/Types/ServerError';
+import { Wrapper } from '../../Styles/SidesheetWrapper';
+import { ReleaseControlErrorBanner } from './ErrorBanner';
+import { HTSidesheet } from '../../Types/pipetest';
+import { Tabs } from '@equinor/eds-core-react';
+import { CheckListTable } from './CheckListTable';
+import { SidesheetTabList } from './SidesheetTabs';
+import { ElectroView } from '../Electro/ElectroView';
+import { useQuery } from 'react-query';
+import { useLocationKey } from '../../../../packages/Filter/Hooks/useLocationKey';
```
Comment (jaysencpp) [inline src/apps/DisciplineReleaseControl/Components/Sidesheet/ReleaseControlHTSidesheet.tsx:11]: useLocationKey fra filter pakken?

File: src/apps/DisciplineReleaseControl/Components/Sidesheet/ReleaseControlSidesheet.tsx [hunk 3]
```diff
@@ -1,22 +1,42 @@
...
 import { useEffect, useState } from 'react';
 import { useQuery } from 'react-query';
 import { useLocationKey } from '../../../../packages/Filter/Hooks/useLocationKey';
 import { ServerError } from '../../Api/Types/ServerError';
 import { fetchAndChewPipetestDataFromApi } from '../../Functions/statusHelpers';
 import { Wrapper } from '../../Styles/SidesheetWrapper';
-import { Pipetest } from '../../Types/pipetest';
 import { ElectroView } from '../Electro/ElectroView';
+import { HTSidesheet, Pipetest } from '../../Types/pipetest';
+// import { Viewer } from '../../../../packages/ModelViewer/ModelViewer';
```
Comment (jaysencpp) [inline src/apps/DisciplineReleaseControl/Components/Sidesheet/ReleaseControlSidesheet.tsx:12]: kan fjerne de importene som er kommentert ut

File: src/apps/DisciplineReleaseControl/Components/Sidesheet/ReleaseControlHTSidesheet.tsx (hunk 2 above)
Comment (jaysencpp) [reply src/apps/DisciplineReleaseControl/Components/Sidesheet/ReleaseControlHTSidesheet.tsx:11]: kunne du ha flyttet useLocationKey hooken til Utils pakken?

File: src/apps/DisciplineReleaseControl/Components/Garden/ReleaseControlGardenGroupView.tsx (hunk 1 above)
Comment (Andreas-Tonnessen) [reply src/apps/DisciplineReleaseControl/Components/Garden/ReleaseControlGardenGroupView.tsx:22]: Ja, egentlig. CustomGroupView er av type Pipetest, men det er ikke det vi sender videre på onSelect.

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**